sentinel.go

```diff
@@ -36,6 +36,10 @@ type FailoverOptions struct {
 	// Route all commands to slave read-only nodes.
 	SlaveOnly bool
 
+	// Allow to access slaves disconnected with master when cannot get connected slaves
+	// Now, this option only works in RandomSlaveAddr function.
+	AllowDisconnectedSlaves bool
```

Comment:
Allow is a bit vague here. I would rename to `UseDisconnectedSlaves`.

Also perhaps disconnected slaves should be used only when there are no healthy slaves?

Reply:
It is renamed to UseDisconnectedSlaves.